### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nericksen**
The example seems to contain a field that is no longer relevant/available for updating custom fields

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShacharKidor** [comment]
Hi @nericksen, we appreciate your effort on contributing to the XSOAR marketplace. Due to excessive amount of contributions in our queue, it may take us a bit more time than usual to review this pull request, we appreciate your patience.

File: Packs/Jira/Integrations/JiraV2/README.md [hunk 1]
```diff
@@ -394,7 +394,7 @@ Modifies an issue in Jira.
 
 
 ##### Command Example
```
Comment (ShacharKidor) [inline Packs/Jira/Integrations/JiraV2/README.md:396]: Not sure why this change is needed? Can you please explain?

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Hi @nericksen, 
Thank you for contributing!

I sent you a message on slack to discuss the change you added.

File: Packs/Jira/Integrations/JiraV2/README.md [hunk 2]
```diff
@@ -394,7 +394,7 @@ Modifies an issue in Jira.
...
-```!jira-edit-issue issueId=15572 customFields=Type_of_incident:Malware(Virus,_Ransomware) description="Just a description"```
+```!jira-edit-issue issueId=DEM-4921 issueJson={"fields":{"customfield_100": "testing3"}}```
```
Comment (ShacharKidor) [inline Packs/Jira/Integrations/JiraV2/README.md:397]:
```suggestion
```!jira-edit-issue issueId=DEM-4921 issueJson=`{"fields":{"description":"testing3"}}````
```

**Comment by nericksen** [comment]
@ShacharKidor ShacharKidor as discussed offline, changes have now been pushed. Let me know if anything else is needed. Thanks again for the feedback!

**Comment by ShacharKidor** [comment]
> @ShacharKidor ShacharKidor as discussed offline, changes have now been pushed. Let me know if anything else is needed. Thanks again for the feedback!

@nericksen - Thank you for pushing the changes, I'll merge your PR.

**Review by ShacharKidor — APPROVED** [review]

**Pull request merged**